lib/pbench/server/auth/auth.py

```diff
@@ -54,7 +54,11 @@ def encode_auth_token(self, time_delta: datetime.timedelta, user_id: int) -> str
 
     def get_secret_key(self):
         try:
-            return os.getenv("SECRET_KEY", "my_precious")
+            return (
+                OpenIDClient.PUBLIC_KEY
+                if OpenIDClient.PUBLIC_KEY
+                else os.getenv("SECRET_KEY", "my_precious")
+            )
```

Comment:
`SECRET_KEY` and `PUBLIC_KEY` are interchangeable??? That's interesting.

Reply:
They are not but from our internal token encode and decode perspective it wouldn't make much difference as things stand right now. And as eventually, we will remove our internal token encode (or at least we won't have it in the current form) I am not worrying too much. But I can see how confusing this can be I will update it.

Reply:
I decided not to update this function at this time as this will eventually be gone away and to avoid confusion.